packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1147,18 +1147,29 @@ class _TabBarState extends State<TabBar> {
     );
   }
 
+  bool _debugScheduleCheckHasValidTabsCount() {
+    WidgetsBinding.instance.addPostFrameCallback((Duration duration) {
```

Comment:
if the `TabBarView` widget is in a `LayoutBuilder` and the `DefaultTabBarController` is not, `build` will be called twice in the same frame and the post frame callback will be scheduled twice.

Reply:
That would mean the error will be thrown twice if the length is indeed inconsistent when the tree is finalized.

Reply:
Fixed.